src/android/io/sqlc/SQLitePlugin.java

```diff
@@ -361,8 +367,10 @@ private boolean deleteDatabaseNow(String dbname) {
 
         public void run() {
             try {
+                logger.logWarning("Opening database with key: " + this.dbkey, "SQLite");
```

Comment:
Not sure if we should be storing the dbkey directly (maybe a hash of it?). We don't need the key, just to know if it changed). Also, I would change the message to "Opening database (hash)". Having "Opening database with key" can alarm a security focused customer when he opens ServiceCenter

Reply:
Done